vitrina/datasets/migrations/0032_contact_contact_name_contact_organization_and_more.py

```diff
@@ -0,0 +1,81 @@
+# Generated by Django 4.2.23 on 2025-11-10 11:25
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+def fill_organization_id(apps, schema_editor):
+    Contact = apps.get_model("vitrina_datasets", "Contact")
+
+    for contact in Contact.objects.all():
+        if contact.dataset:
+            contact.organization = contact.dataset.organization
+            contact.save()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("vitrina_orgs", "0009_alter_publisherorganization_options"),
+        ("contenttypes", "0002_remove_content_type_name"),
+        ("vitrina_datasets", "0031_auto_20251027_0636"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="contact",
+            name="contact_name",
+            field=models.CharField(blank=True, max_length=255, verbose_name="Vardas"),
+        ),
+        migrations.AddField(
+            model_name="contact",
+            name="organization",
+            field=models.ForeignKey(
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                to="vitrina_orgs.organization",
+                verbose_name="Organizacija",
+            ),
+        ),
+        migrations.AddField(
+            model_name="contact",
+            name="position",
+            field=models.CharField(blank=True, max_length=255, verbose_name="Pareigos"),
+        ),
+        migrations.AlterField(
+            model_name="contact",
+            name="content_type",
+            field=models.ForeignKey(
+                limit_choices_to={"model__in": ("organization", "user")},
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                to="contenttypes.contenttype",
+                verbose_name="Content Type",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="contact",
+            name="dataset",
+            field=models.ForeignKey(
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                to="vitrina_datasets.dataset",
+                verbose_name="Duomenų rinkinys",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="contact",
+            name="object_id",
+            field=models.PositiveIntegerField(null=True, verbose_name="Object ID"),
+        ),
+        migrations.RunPython(fill_organization_id), 
+        migrations.AlterField(
+            model_name="contact",
+            name="organization",
+            field=models.ForeignKey(
+                on_delete=django.db.models.deletion.CASCADE,
+                to="vitrina_orgs.organization",
+                verbose_name="Organizacija",
+            ),
+        ),
```

Comment:
If I understand correctly, you added a new field as `null=True`, filled it with values and then removed the `null=True`? Is it ok to leave it in a single migration file? I remember it was causing me errors, when I needed to add a new non-nullable field for already existing model with objects.